docs/releases/v2018.2.1.rst

```diff
@@ -0,0 +1,91 @@
+Gluon 2018.2.1
+==============
+
+Bugfixes
+~~~~~~~~
+
+* Fixes a bug in the batman-adv respondd module that caused duplicate
+  IPv6 addresses in nodeinfo replies (`#1615 <https://github.com/freifunk-gluon/gluon/issues/1615>`_)
+
+  This was visible on the status page and several map implementations.
+  The new implementations uses netlink instead of parsing `/proc/net/if_inet6`.
+
+* Fixes a localization issue in gluon-config-mode-geo-location which 
+  resulted in a partial translation of the node locations description 
+  text. (`#1611 <https://github.com/freifunk-gluon/gluon/issues/1611>`_)
+
+* Fixes the display of the improved memory usage estimation in gluon-status-page
+
+  This change was actually already merged in time for v2018.2 but the 
+  javascript was not rebuilt.
+
+* Fixes automatic updates for several devices by adding and updating 
+  the autoupdater image names
+
+  This affects the following device:
+
+  * GL.iNet GL-AR150, 
+  * GL.iNet GL-AR300M
+  * GL.iNet GL-AR750
+  * Raspberry Pi Model B+ Rev 1.2
+
+* Fixes the primary MAC address selection for Unifi AC 
+  Lite/Mesh/Pro/Mesh Pro (`#1629 <https://github.com/freifunk-gluon/gluon/issues/1629>`_)
+
+* Fixes low data rate selection for multicast and management frames on
+  ath10k and ath10k-ct (`#1644 <https://github.com/freifunk-gluon/gluon/pull/1644>`_)
+
+  A patchset has been backported that notifies these drivers of requested data rate changes
+
+* Fixes the data rate selection in ath10k and ath10k-ct when no 
+  `mcast_rate` is configured (`#1657 <https://github.com/freifunk-gluon/gluon/pull/1657>`_)
+
+  Previously a missing mcast_rate could result in broken 5 GHz connectivity
+
+New features
+~~~~~~~~~~~~
+
+Scheduled domain switch
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Gluon has support for multiple domains since it's v2018.2 release.
+The scheduled domain switch allows for reliable migrations between 
+domains at a preconfigured time.
```

Comment:
... This can be useful for Communities who want to switch wireless protocols (IBSS -> 802.11s) or activate VXLAN encapsulation on wired mesh links.